src/controllers/user.controller.ts

```diff
@@ -0,0 +1,72 @@
+import { dataSource } from './../configs/dbConfig';
+import { Request, Response } from 'express';
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcrypt';
+import User from '../entity/user.entity';
+import { FindOneOptions } from 'typeorm';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const JWT_SECRET = process.env.JWT_SECRET!;
+
+export const register = async (req: Request, res: Response) => {
+  try {
+    const { email, password, contact_email, first_name, last_name, image_url, linkedin_url } = req.body;
+
+    if (!email || !password || !contact_email || !first_name || !last_name || !image_url || !linkedin_url) {
```

Comment:
Only these are required. @sathsarabandaraj, set null to the other values

```suggestion
    if (!email || !password) {
```